### PR TITLE
React rendering lists

### PR DESCRIPTION
**Pull request by jghil**
image

### CODE REVIEW

File: react-rendering-lists/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+function PokemonList(props) {
+  const pokedex = [
+    { number: '001', name: 'Bulbasaur' },
+    { number: '004', name: 'Charmander' },
+    { number: '007', name: 'Squirtle' },
+    { number: '025', name: 'Pikachu' },
+    { number: '039', name: 'Jigglypuff' }
+  ];
+  const pokemonNames = pokedex.map(pokemon =>
+    <li key={pokemon.name}>{pokemon.name}</li>
```
Comment (scbowler) [inline react-rendering-lists/src/index.jsx]: The `key` should use the `number` property:
```JSX
<li key={pokemon.number}>{pokemon.name}</li>
```

File: react-rendering-lists/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  return (
+    <ul>
+      {pokemonNames}
+    </ul>
+  );
+}
+
+const root = ReactDOM.createRoot(document.querySelector('#root'));
+root.render(<PokemonList pokedex={name} />);
```
Comment (scbowler) [inline react-rendering-lists/src/index.jsx]: No need to pass a `prop` to `PokemonList` here.
```JSX
root.render(<PokemonList />);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but a couple updates needed. Let me know if you have any questions.

File: react-rendering-lists/src/index.jsx (hunk 1 above)
Comment (jghil) [reply react-rendering-lists/src/index.jsx]: changes made!

File: react-rendering-lists/src/index.jsx (hunk 2 above)
Comment (jghil) [reply react-rendering-lists/src/index.jsx]: changes made!

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**